aspnetcore/fundamentals/index.md

```diff
@@ -264,6 +264,8 @@ For more information, see [Content root](xref:fundamentals/host/web-host#content
 
 The web root (also known as *webroot*) is the base path to public, static resources, such as CSS, JavaScript, and image files. The static files middleware will only serve files from the web root directory (and sub-directories) by default. The web root path defaults to *{Content Root}/wwwroot*, but a different location can be specified when [building the host](#host).
 
+For more information, see [Web root](xref:fundamentals/host/web-host#webroot).
```

Comment:
This close to 3.0 GA, I would link to generic-host rather than soon-to-be-deprecated web-host.  You'll have to use a regular relative link that specifies 3.0.

Reply:
Thanks Tom!  I will add a moniker range to point to ContentRootPath on the .NET Generic Host doc for 3.0 and point to the Web root for earlier versions.